components/dialog/test/dialog.test.js

```diff
@@ -37,6 +37,27 @@ describe('d2l-dialog', () => {
 			expect(getComposedActiveElement()).to.equal(button);
 		});
 
+		it('should focus on an autofocus element', async() => {
+			const el = await fixture(html`<d2l-dialog opened><p autofocus tabindex="-1">focus</p><button>focus</button></d2l-dialog>`);
+			const paragraph = el.querySelector('p');
+			await oneEvent(el, 'd2l-dialog-open');
+			expect(getComposedActiveElement()).to.equal(paragraph);
```

Comment:
Do these tests actually pass? If so, we could unskip the parent and just add `it.skip` to the ones that are broken.